pallets/xcm-transactor/src/lib.rs

```diff
@@ -236,14 +237,14 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			dest: T::Transactor,
 			index: u16,
-			fee_location: VersionedMultiLocation,
```

Comment:
why ?

Reply:
@girazoki

Reply:
Sorry I didn't saw the surrounding context. Yeah, this is because the Call enum is greater than 1024 bytes without a boxed `VersionMultilocation`. Check [here](https://github.com/paritytech/substrate/blob/master/frame/utility/src/lib.rs#L142-L152). Is actually handled the same way in polkadot.